TaskVlopperTests/TaskVlopperTests.csproj

```diff
@@ -0,0 +1,90 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why, O, why do we need another test project in solution? I simply don't get that. Error or intended?